src/platform/Linux/BUILD.gn

```diff
@@ -74,6 +74,9 @@ static_library("Linux") {
     "PosixConfig.h",
     "SystemPlatformConfig.h",
     "SystemTimeSupport.cpp",
+    "WiFiPAFDriver.h",
+    "WiFiPAFDriver.cpp",
```

Comment:
Should this be conditional on some "enable WiFi PAF" setting instead? Do we expect every linux distribution to support this?

Reply:
I agree, this should be guarded somehow. Will fix it.